README.md

```diff
@@ -52,3 +52,29 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+#Testing the App
+Unit Tests written for app. Tests to be run using command
+$ poetry run pytest
+
+# Running/building app via Docker use the following commands in Prod/dev environments
+# Note local .env file is passed in during Docker run.
+
+# running in Production env 
+docker build --target production --tag todo-app:prod .
+
+docker run --env-file ./.env -p 5001:5000 --mount "type=bind,source=$(pwd)/todo_app,target=/opt/todo_app" todo-app:prod
+# http://127.0.0.1:8000/ to run 
+
+# Running in Development env
+docker build --target development --tag todo-app:dev .
+
+docker run --env-file ./.env -p 5001:5000 --mount "type=bind,source=$(pwd)/todo_app,target=/opt/todo_app" todo-app:dev
```

Comment:
When I test out the hot-reloading here (which should be enabled by your bind-mount), I don't see the app changing live so I think this misses your code slightly. Looking closer, the reason is that we've got a slight directory mismatch:
* Your Dockerfile sets a `WORKDIR` of `/opt/todoapp`
* You then copy the `todo_app` folder into there, so that it lives at `/opt/todoapp/todo_app`
* But this bind-mount targets `todo_app` into `/opt/todoapp` so just slightly mis-synced

So to fix, I think we want:
```suggestion
docker run --env-file ./.env -p 5001:5000 --mount "type=bind,source=$(pwd)/todo_app,target=/opt/todoapp/todo_app" todo-app:dev
```